app/controllers/brexit_landing_page_controller.rb

```diff
@@ -1,16 +1,28 @@
 class BrexitLandingPageController < ApplicationController
+  include BrexitLandingPageTestable
+
   def show
     setup_content_item_and_navigation_helpers(taxon)
 
-    render locals: {
-      presented_taxon: presented_taxon,
-      presentable_section_items: presentable_section_items,
-      show_dynamic_list: show_dynamic_list?,
-    }
+    if page_variant.variant?("A")
+      render "show_A", view_locales
+    else
+      render "show_B", view_locales
+    end
```

Comment:
It's unconventional to use capital letters in the view names, I'd suggest lowercasing them.

Also it might be overkill having the view_locales in a method when it'd be a bit of a simpler ready if they were set before the conditional (as you have to go off to mentally break off to read the method to understand the data)

You may even want to go along the lines of:

```
view = page_variant.variant?("A") ? "show_a" : "show_b"

render view, locals: { ... }
```

Also will this default to A if CDN sets nothing? I'm guessing so based on the control group part.

Reply:
Sure, I like your suggestion! You're correct, the default will indeed by variant 'A' if no header is set from the CDN.